web/src/add_group_members_pill.ts

```diff
@@ -136,6 +167,10 @@ export function create_without_add_button({
         onPillRemoveAction(get_pill_user_ids(pill_widget), get_pill_group_ids(pill_widget));
     });
 
+    pill_widget.onPillExpand((pill) => {
+        expand_group_pill(pill, pill_widget);
+    });
+
     add_subscribers_pill.set_up_pill_typeahead({
```

Comment:
It strikes me that `create_without_add_button` probably should just be an option to `create`? Can be a follow-up.